fuse/dl/lightning/pl_module.py

```diff
@@ -195,13 +195,15 @@ def validation_step(
         else:
             losses = self._losses
         # given the batch_dict and FuseMedML style losses - compute the losses, return the total loss (ignored) and save losses values in batch_dict["losses"]
-        _ = step_losses(losses, batch_dict)
+        if losses is not None:
```

Comment:
How `losses` can be `None`? And why the same change shouldn't be in `train` as well?

I thought that we must define one of the both: `self._validation_losses`, `self._losses`. 

Also we have the line:
```python 
        self._losses = losses if losses is not None else {}
```

So maybe we can do the same for `_validation_losses` (or assert that one of them exists?)

Reply:
Just throwing thoughts.. I'm not familiar with the motivation :)

Reply:
loss is a must to train a model (and not for validation).
The code is confusing right now, let me make it clearer. Thanks for the feedback!